### PR TITLE
Explain that pull requests may be rejected

### PR DESCRIPTION
**Pull request by vinc456**
Contributing to Open Source Open Space

### CODE REVIEW

**Comment by sims1253** [comment]
Hey there,
You might want to take a look at our commit message guide here:
http://coala.readthedocs.io/en/latest/Getting_Involved/Writing_Good_Commits.html

File: docs/Users/Tutorials/Git_Help.rst [hunk 1]
```diff
@@ -124,8 +124,11 @@ Then type:
 
 This will lead you to a text editor. Now you need to write your commit message.
 We are very strict about writing commit messages as they help us maintain
-*coala* **clean** and **stable**. Commit messages usually consists of three main
-parts. They should have a newline between them.
+*coala* **clean** and **stable**. Commit messages usually consists of three
+main parts. They should have a newline between them. We'll try to offer some
+suggestions for improvement and you can try to commit again after making the
+relevant changes. Sometimes it may take several interations of going back
+and forth before the changes will be accepted.
```
Comment (Makman2) [inline docs/Users/Tutorials/Git_Help.rst]: oh sorry just seeing, would you mind to link the user name + email of the commit to the actual github id?
you can do that by typing
```
git config user.name your-github-name
git config user.email your-github-email
git commit --amend --reset-author
git push <remote> master --force
```
:)

**Comment by Makman2** [comment]
hmm still didn't work, are you sure that the email address on this commit and from your github account are the same?

**Comment by sils1297** [comment]
Hi, this needs a rebase, the commit message needs work and your author metadata isn't fully set up yet.

Rebase: [This page](http://coala.readthedocs.org/en/latest/Users/Tutorials/Git_Help.html) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

Ping us if you need any further help!

18cd9e9 needs work

**Comment by sils1297** [comment]
hey, great! A few nitpicks - please be aware that I'm just being very very picky, this is already great stuff.

1. I'd remove the `rst` from the context tag as it doesn't add information
2. The shortlog lacks a verb, you usually want one :)

**Comment by sils1297** [comment]
I have no idea how to get a proper verb there though :P maybe this phrasing isn't so bad... not sure

**Comment by Makman2** [comment]
you can take "may" as some kind of verb, imo the commit message is okay. I think the .rst doesn't need to be removed either, so we can better distinct from python files (it doesn't matter really but anyway :+1: )
though the github ID is still not linked to the commit :3

**Comment by Makman2** [comment]
and you need again a rebase :)

**Comment by sils1297** [comment]
Hi,

do you need help figuring this out? We're all happy to help you on [our gitter channel](https://gitter.im/coala-analyzer/coala), just drop by and ask us anything :)

**Comment by Makman2** [comment]
ping @vinc456 :)

**Comment by vinc456** [comment]
Hello yes I could use a hand with the rebase. I'll hop onto Gitter after Thursday and wrap this up once and for all.

**Comment by Makman2** [comment]
cool, then see you on thursday :D

**Comment by sils1297** [comment]
Hey, sorry for taking so long here. Some problems:

First: instead of `Fixes`, please use `Closes`, we are distinguishing this by now because we will use Fixes for some fancy bug prediction in the future.

Second, as pointed out by @Makman2 you may want to check if your commit is attributed correctly in GitHub. We can merge it anyway but if you care for it to appear correctly on your profile you'll want to check that. (GitHub shows it's unattributed ATM.)

Otherwise it's good, rebase, amend and we can merge it :)

**Comment by Adrianzatreanu** [comment]
hey, this is a strong WIP. 
first of all, you created 3 commits that do the same thing. could you remove 2 of them and leave one only? to do this you can type ``git pull origin master`` first to get your branch even with our master branch, and then type ``git rebase -i origin`` and simply erase 2 of the 3 lines until you are left with only one commit. and then you can push again :)

**Comment by Adrianzatreanu** [comment]
just contact us on https://gitter.im/coala-analyzer/coala and we'd love to help you :)

**Comment by Makman2** [comment]
@vinc456 if you have nothing against it, I would take this up so we get it merged :)
don't worry I keep the author information :)

**Comment by Makman2** [comment]
https://github.com/coala-analyzer/coala/pull/2637 takes over.

**Comment by vinc456** [comment]
Sorry about that, I was busy making some deadlines. 

This might be useful for someone else in the future. [On undoing, fixing, or removing commits in git](http://sethrobertson.github.io/GitFixUm/fixup.html)

**Comment by Makman2** [comment]
no problem :D

**Pull request closed without merging**